bundles/plugins/org.bonitasoft.studio.designer/messages.properties

```diff
@@ -38,7 +38,7 @@ openUiDesignerInformationToggleMessage=Do not show this dialog again.
 hideEmptyContractDialogTitle=Empty contract
 hideEmptyContractDialogMessage=This button generates a form with the relevant widgets and data bindings for the contract inputs. The mapping between the form is and this (task or process => 0) is done once for all. It is not updated when you update the contract. We recommend you create your contract before clicking on this button.\nDo you still want to open the UI Designer?
 hideEmptyContractDialogToggleMessage=Do not show this dialog again.
-waitingForTomcatServer=Tomcat server is booting in background (this may take a few seconds)...
+waitingForTomcatServer=The Tomcat server is starting in the background. This takes a few seconds.\nThe UI Designer will open when Tomcat has started.\nClick Cancel to close this popup.
```

Comment:
Did you try to click on Cancel? Is it working?
There should be another message to reuse for UI Designer.
Are you sure that this message is used only when opening the UI designer? i think that it is not the case